views/random-beers.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="randomBeer">
```

Comment:
Could this page be styled according to the example in the iteration?

Reply:
I'll fix the style.